pretty_assertions/Cargo.toml

```
@@ -27,3 +27,7 @@ diff = "0.1.12"
 [target.'cfg(windows)'.dependencies]
 output_vt100 = "0.1.2"
 ctor = "0.1.9"
+
+[features]
+default = []
+no_std = []
```

In rust features should be additive. See [Rust API Guidelines](https://rust-lang.github.io/api-guidelines/naming.html#feature-names-are-free-of-placeholder-words-c-feature)

> As a related note, Cargo requires that features are additive so a feature named negatively like no-abc is practically never correct.

```suggestion
std = []
```

---

Thanks - updated, and added the alloc feature as you suggested.